config/example-extras.cfg

```diff
@@ -630,6 +630,13 @@
 #   measurements must be provided.
 
 
+# Filament runnout sensor. This will enable a filament runnout sensor
```

Comment:
Very minor here, just a correction of spelling and grammar, runnout should be run out.